template/thumbnails_css_js.tpl

```diff
@@ -108,9 +117,13 @@ $cdm.on('click', '.add, .remove', function(e) {
           
           if (method == 'pwg.collections.removeImages') {
             col_ids.splice(col_ids.indexOf(col_id), 1);
+            if (col_ids == ''){
```

Comment:
isn't `col_ids` an array ? emptiness should be tested with `col_ids.length`

The fact that in JS `[] ==''` should not be exploited. (this is a whole bunch of old code but double equal must never be used, it's really error-prone)